crashreport/src/main/java/com/rakuten/tech/mobile/crash/processors/CustomKeyCache.java

```diff
@@ -62,7 +57,7 @@ public void addCustomKey(String key, String value)
    */
   public void removeCustomKey(String key) {
     // Removes an existing key from the custom key map.
-    if (key != null && customKeysMap.containsKey(key)) {
```

Comment:
Just making a note for when we polish the public API:

Maybe this should return the value of `HashMap#remove` so they can know if the key existed?